go.sum

```diff
@@ -39,15 +35,22 @@ github.com/hashicorp/golang-lru v0.5.1 h1:0hERBMJE1eitiLkihrMvRVBYAkpHzc/J3QdDN+
 github.com/hashicorp/golang-lru v0.5.1/go.mod h1:/m3WP610KZHVQ1SGc6re/UDhFvYD7pJ4Ao+sR/qLZy8=
 github.com/hpcloud/tail v1.0.0 h1:nfCOvKYfkgYP8hkirhJocXT2+zOD8yUNjXaWfTlyFKI=
 github.com/hpcloud/tail v1.0.0/go.mod h1:ab1qPbhIpdTxEkNHXyeSf5vhxWSCs/tWer42PpOxQnU=
+github.com/iredelmeier/lightstep-tracer-common/golang/gogo v0.0.0-20200214020106-93e2f3cec85e h1:YiQ+VsjS97Q935VOAOyOw76V7RN+QO5V2aCKRrv0ihc=
```

Comment:
Not sure why references to my fork are still in here. Maybe try `go mod tidy`?

Reply:
Fixed